ledger/complete/mtrie/trie/trie_test.go

```diff
@@ -505,7 +506,7 @@ func Test_Pruning(t *testing.T) {
 	})
 
 	t.Run("smoke testing trie pruning", func(t *testing.T) {
-		t.Skip("skipping trie pruning smoke testing as its not needed to always run")
+		unittest.SkipUnless(t, unittest.TEST_LONG_RUNNING, "skipping trie pruning smoke testing as its not needed to always run")
```

Comment:
@ramtinms does this capture the skip reason correctly? Or should we introduce a new category for this?

Reply:
thats a good reason